smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -1062,6 +1064,11 @@ impl<'de> SerdeDeserialize<'de> for Duration {
     }
 }
 
+/// Canonical address of an account, as raw bytes.
+#[derive(Eq, PartialEq, Copy, Clone, PartialOrd, Ord, Debug, Hash)]
```

Comment:
Since this is just a wrapper type we should mark it `transparent`:
```suggestion
#[derive(Eq, PartialEq, Copy, Clone, PartialOrd, Ord, Debug, Hash)]
#[repr(transparent)]
```